source/module_hsolver/src/cuda/dngvd_op.cu

```diff
@@ -289,38 +289,28 @@ void dngvd_op<double, psi::DEVICE_GPU>::operator()(const psi::DEVICE_GPU* d,
                                                    std::complex<double>* V)
 {
     assert(nstart == ldh);
-    // init A_eigenvectors & transpose_B
-    double2 *A_eigenvectors, *transpose_B;
-    checkCudaErrors(cudaMalloc((void**)&A_eigenvectors, sizeof(double2) * ldh * nstart));
-    checkCudaErrors(cudaMalloc((void**)&transpose_B, sizeof(double2) * ldh * nstart));
-
-    // transpose A, B  to A_eigenvectors, transpose_B
-    matrixTranspose_op<double, psi::DEVICE_GPU>()(d, ldh, nstart, A, (std::complex<double>*)A_eigenvectors);
-    matrixTranspose_op<double, psi::DEVICE_GPU>()(d, ldh, nstart, B, (std::complex<double>*)transpose_B);
 
-    // init all_W
-    double* all_W;
-    checkCudaErrors(cudaMalloc((void**)&all_W, sizeof(double) * ldh));
+    // A to V
+    checkCudaErrors(cudaMemcpy(V, A, sizeof(double2) * ldh * nstart, cudaMemcpyDeviceToDevice));
 
     // prepare some values for cusolverDnZhegvd_bufferSize
-    cusolverDnHandle_t cusolverH;
-    cusolverErrcheck(cusolverDnCreate(&cusolverH));
     int* devInfo;
     checkCudaErrors(cudaMalloc((void**)&devInfo, sizeof(int)));
-
+    cusolverDnHandle_t cusolverH;
+    cusolverErrcheck(cusolverDnCreate(&cusolverH));
```

Comment:
Is it OK to create cusolver handle each time? This process may use some additional time.